meson.build

```diff
@@ -2572,6 +2589,48 @@ endforeach
 
 ############################################################
 
+prev = ''
+foreach p : fuzz_regression_tests
+        a = p.split('/')[-3]
+        b = p.split('/')[-2]
+        c = p.split('/')[-1]
+
+        if a == 'address'
+                build = sanitize_address
+        else
+                error('unknown sanitizer @0@'.format(a))
+        endif
+
+        name = '@1@:@0@'.format(a, b)
+
+        if name != prev
+                if want_tests == 'false'
+                        message('Not compiling @0@ because tests is set to false'.format(name))
+                elif not sanitizers.contains(a)
+                        message('Not compiling @0@ because @1@ sanitizer is not available'.format(name, a))
+                elif slow_tests
+                        exe = custom_target(
+                                name,
+                                output : name,
+                                depends : build,
+                                command : [env,
+                                           'ln', '-fs',
+                                           join_paths(build.full_path(), b),
```

Comment:
While `meson-0.44.1` is able to successfully build `systemd` when `-Dslow-tests=true` is passed, `meson-0.42.1` fails with the following error:
```sh
Meson encountered an error in file meson.build, line 2619, column 16:
Command line arguments must be strings, files or targets.
```
. Should `meson_version` at the top of `meson.build` be bumped? Or could it be that the error is the result of something else?

Reply:
Dunno. Can you debug it and come up with a version that works with old meson?

Reply:
`git bisect` says that the commit https://github.com/mesonbuild/meson/pull/2375/commits/bb0e18b73885de374f8461c0e4f3c911fded1e46 makes everything work, so the easiest solution would be to use `'meson_version : '>= 0.43'`. I'm not sure that I will be able to get the bug around in a timely manner, but probably @nirbheek might help.

Reply:
That's tough. In Debian everything up to testing has meson-0.42. I'll try to make this work with older meson.